root/types.js

```diff
@@ -57,6 +57,15 @@ declare type OptionTreeT = {|
   +description: string,
 |};
 
+declare type WorkT = EntityT & {
```

Comment:
All the types you've added here should be in alphabetical order, as noted in the comment at the top of the file. ;)